terraform/staff-device-management-virtualdesktop.tf

```diff
@@ -32,5 +32,15 @@ module "staff-device-management-virtualdesktop" {
       added_by     = "matthew.white1@justice.gov.uk" #  Who made the decision to grant them access? e.g. 'Awesome Team <awesome.team@digital.justice.gov.uk>'
       review_after = "2021-12-31"                    #  Date after which this grant should be reviewed/revoked, e.g. 2021-11-26
     },
+    {
+      github_user  = "jabenny"
+      permission   = "push"
+      name         = "Josh Benfield"                          #  The name of the person behind github_user
+      email        = "jbenfield@sol-tec.com"                  #  Their email address
+      org          = "Sol-Tec"                                #  The organisation/entity they belong to
+      reason       = "PTTP Tech Team"                         #  Why is this person being granted access?
+      added_by     = "richard.baguley@digital.justice.gov.uk" #  Who made the decision to grant them access? e.g. 'Awesome Team <awesome.team@digital.justice.gov.uk>'
```

Comment:
The format of this needs to match the example:
```
Richard Baguley <richard.baguley@digital.justice.gov.uk>
```
Also, if a team email address is available, that would be better (in case the named individual is not available when this collaboration comes up for review).